index.html

```diff
@@ -9,6 +9,7 @@
     <title>Awesome Books | Home</title>
   </head>
   <body>
+    <!-- ============================== Header ============================== -->
     <header>
       <h4>Awesome Books</h4>
       <div id="datetime"></div>
```

Comment:
Great job @aamirkhan2478 🤝

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md), kindly display the date and time below the navigation bar.
- 
![date](https://user-images.githubusercontent.com/63917725/183458198-7fb77684-9d8e-4788-ac82-561187121b2f.PNG)